src/common/perf_counters.h

```diff
@@ -233,6 +233,9 @@ class PerfCounters
 
   ~PerfCounters();
 
+  bool time_filtered = false;
+  std::atomic<ceph::coarse_real_clock::duration> time_alive = { std::chrono::nanoseconds::zero() };
+  ceph::coarse_real_clock::time_point last_updated;
```

Comment:
`last_updated` is the one that needs atomic, not `time_alive`

Reply:
Fixed!